lib/buildpackutil.py

```diff
@@ -328,8 +328,8 @@ def ensure_and_get_mono(mx_version, cache_dir):
 
 
 def _determine_jdk(mx_version, package="jdk"):
-    oracle_jdks = ["7", "8u51", "8"]
-    adoptopenjdk_jdks = ["8u202"]
+    oracle_jdks = ("7", "8u51", "8")
+    adoptopenjdk_jdks = "8u202"
```

Comment:
You changed this from a `list` with one element into simply a string, so your check on line `337`, `elif java version in adoptopenjdk_jdks` will not work as intended.